Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -4521,6 +4521,11 @@ private bool CheckForGameEnd(bool checkbitten = false)
                         }
                     }
                     break;
+                case 3:
+                case 4:
```

Comment:
This is questionable. Do we really want to end a game at 3 or 4 players? The tanner (if there is one) could still win. So the game should never end at 4 players and only end at 3 players if there is no tanner among them.